pkg/importer/dataStream.go

```diff
@@ -324,121 +306,73 @@ func (d *DataStream) http() (io.ReadCloser, error) {
 	return countingReader, nil
 }
 
-//registryData - temporary data used by import from registry flow
-//is deleted as part of dataStream.Close()
-type registryData struct {
-	dataDir string
```

Comment:
This is temporary directory where registry flow stores its data, it should just point to tmpPvc location. This is a data context which cleanup is called on datastream.Close(). So all registry related temprary data is removed by the end of the flow.